core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -158,7 +159,15 @@
 
     public SystemSessionProperties()
     {
-        this(new QueryManagerConfig(), new TaskManagerConfig(), new MemoryManagerConfig(), new FeaturesConfig(), new NodeMemoryConfig(), new DynamicFilterConfig(), new NodeSchedulerConfig());
+        this(
+                new QueryManagerConfig(),
+                new TaskManagerConfig(),
+                new MemoryManagerConfig(),
+                new FeaturesConfig(),
+                new NodeMemoryConfig(),
+                new DynamicFilterConfig(),
+                new NodeSchedulerConfig(),
+                new OptimizerConfig());
```

Comment:
nit: i'd add this after FeaturesConfig, since they remain related

Reply:
done